src/css/components/_big-play.scss

```diff
@@ -17,6 +19,17 @@
   @include transition(all 0.4s);
 
   @extend .vjs-icon-play;
+
+  // Since the big play button doesn't inherit from vjs-control, we need to specify a bit more than
```

Comment:
We should think about how we can remove this duplication with a shared class or something.